static/js/src/entry/donate/Thermometer.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="thermometer">
+    <div class="bar">
+      <div class="bar_inner bar_inner--goal"></div>
+      <div class="bar_inner bar_inner--actual" v-bind:style="{width: actualWidth + '%'}"></div>
```

Comment:
`v-bind:style` can be shortened to just `:style`. Also, I'd use template literals instead of string concatenation for `actualWidth + '%'`.